.github/workflows/node-release.yml

```diff
@@ -8,7 +8,10 @@ on:
         required: true
         type: choice
         options:
-          - prerelease 
+          - prerelease
```

Comment:
the prerelease doesn't have to be here anymore

Reply:
In testing I found you still need it after a prepatch, preminor, or premajor

Say you do premajor and get v6.0.0-pre0. 

If you do a prepatch from this release you get v6.0.1-pre0. 
If you do a prerelease from this releaes you get 6.0.0-pre1

Reply:
okay, so that's how it's operator

Reply:
ya, honestly I had no idea how the npm versioning worked until I tried it.

If you are at v6.0.0-pre0 and do another premajor, it brings you up to v7.0.0-pre0